config/fedora.yml

```diff
@@ -0,0 +1,4 @@
+user: fedoraAdmin
```

Comment:
There's some duplication of work here. I created an issue for this config stuff here: #47 which I am currently working on.

Also I rather not have a yaml file for each piece of infrastructure we are using as that will be 4+ yaml files. (redis/solr/fedora/swift/etc). Why not just put everything in one yaml file?